resources/views/pages/achievement/[achievement]/report-issue.blade.php

```diff
@@ -98,64 +98,88 @@
+        >
             I unlocked this achievement without meeting the requirements.
         </x-ticket.guide-link>
     @endif
 
     @php
-        $subject = urlencode("Typo: $achievement->title ({$achievement->game->title})");
-        $message = urlencode("I'd like to report a spelling/grammar error in [ach=$achievement->id]:\n(Describe the issue here)");
+        $subject = urlencode("Typo: {$achievement->title} [{$achievement->id}] ({$achievement->game->title})");
+        $message = urlencode("I'd like to report a spelling/grammar error in [ach={$achievement->id}]:\n(Describe the issue here)");
     @endphp
-    <x-ticket.guide-link buttonText="Message WritingTeam"
-                         href="{{ route('message.create') }}?to=WritingTeam&subject={{ $subject }}&message={{ $message }}">
+    <x-ticket.guide-link
+        buttonText="Message WritingTeam"
+        href="{{ route('message.create') }}?to=WritingTeam&subject={{ $subject }}&message={{ $message }}&templateKind=writing-error"
+    >
         There is a spelling or grammatical error in the title or description.
     </x-ticket.guide-link>
 
     @php
-        $subject = urlencode("Incorrect type: $achievement->title ({$achievement->game->title})");
-        $message = urlencode("I'd like to report a misclassification error in [ach=$achievement->id]:\n(Describe the issue here)");
+        $subject = urlencode("Incorrect type: {$achievement->title} [{$achievement->id}] ({$achievement->game->title})");
+        $message = urlencode("I'd like to report a misclassification error in [ach={$achievement->id}]:\n(Describe the issue here)");
     @endphp
-    <x-ticket.guide-link buttonText="Message QATeam"
-                         href="{{ route('message.create') }}?to=QATeam&subject={{ $subject }}&message={{ $message }}">
+    <x-ticket.guide-link
+        buttonText="Message QATeam"
+        href="{{ route('message.create') }}?to=QATeam&subject={{ $subject }}&message={{ $message }}&templateKind=misclassification"
+    >
         The achievement type (progression/win/missable) is not correct.
     </x-ticket.guide-link>
 
     @php
-        $subject = urlencode("Unwelcome Concept: $achievement->title ({$achievement->game->title})");
-        $message = urlencode("I'd like to report an unwelcome concept in [ach=$achievement->id]:\n(Describe the issue here)");
+        $subject = urlencode("Unwelcome Concept: {$achievement->title} [{$achievement->id}] ({$achievement->game->title})");
+        $message = urlencode(
+            "I'd like to report an unwelcome concept in [ach={$achievement->id}].\n\n" .
+
+            "- Which Unwelcome Concept:\n" .
+            "(Insert which concept from the docs here)\n\n" .
+
+            "- Detailed Explanation:\n" .
+            "(provide as much detail as possible here. Assume the reader may not have played the game before. The more detail you provide, the better your case.)"
```

Comment:
First word "provide" is still uncapitalized.

Reply:
Looks like both this one was uncapitalized as well as the one for the unwelcome concept prompt. Fixed in latest.